public/components/agents/sca/inventory.tsx

```diff
@@ -577,19 +630,28 @@ export class Inventory extends Component {
         onClick={() => this.setState({ showMoreInfo: !this.state.showMoreInfo })}
       ></EuiButtonEmpty>
     );
+    const { agent, withoutDashboard } = this.props;
+    // console.log(agent, this.state.lookingPolicy, this.state.loading, this.state.redirect, 'perro')
+    // const itemsMapped = this.state.policies.map((item) => {
+    //   return ({
+    //     ...item, pass: <div style={{color:'red'}}>{item.pass}</div>,
+    //     fail: <div>{item.fail}</div>
+    //   }
+    //   )
+    // })
     return (
       <Fragment>
         <div>
-          {this.state.loading && (
+          {this.state.loading || this.state.isLoading && (
             <div style={{ margin: 16 }}>
               <EuiSpacer size="m" />
               <EuiProgress size="xs" color="primary" />
             </div>
           )}
         </div>
         <EuiPage>
```

Comment:
I think replacing the `EuiPage` component with a simple `div` will make it cleaner and gain a few pixels in space without those margins. It would also remove that gray background.

With `EuiPage`
![Screenshot from 2022-10-13 11-30-39](https://user-images.githubusercontent.com/9343732/195560297-7592e8f7-2997-4cda-8120-6833d3732fa3.png)

With a `div`
![Screenshot from 2022-10-13 11-24-58](https://user-images.githubusercontent.com/9343732/195560343-bbaaad55-3345-4674-9810-842dba0f1d5f.png)